File: front/lib/data/models/membership_model/membership.dart
class Membership {
  String? msNo;
  String? title;
  int? period;
  String? contents;
  int? price;

  Membership(this.msNo, this.title, this.period, this.contents, this.price);

  @override
  String toString() {
    return 'Membership{msNo: $msNo, title: $title, period: $period, contents: $contents, price: $price}';
  }

  Membership.fromJson(Map<String, dynamic> json)
      : msNo = json['msNo'],
        title = json['title'],
        period = json['period'],
        contents = json['contents'],
        price = json['price'];
}

File: back/src/main/java/com/pingo/entity/membership/Membership.java
package com.pingo.entity.membership;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class Membership {
    private String msNo;
    private String title;
    private int period;
    private String contents;
    private int price;
}
